device_policy.py

```diff
@@ -44,59 +64,46 @@ def serialize_policy(self, private_key):
 
   def import_policy(self, policy_dict):
     proto_map_path = schema_dir / "device_policy_proto_map.yaml"
-    legacy_map_path = schema_dir / "legacy_device_policy_proto_map.yaml"
     proto_map = yaml.safe_load(proto_map_path.read_bytes())
-    legacy_proto_map = yaml.safe_load(legacy_map_path.read_bytes())
 
-    #get list of device policy keys
-    for key, item in legacy_proto_map.items():
-      if not key or not item:
-        continue
-      proto_map[key] = item
```

Comment:
Argh this code actually serves an important purpose. Don't just remove stuff you don't understand.

Reply:
I removed it to remove support for the legacy policies, but now I know what they were for.